samples/popup/popup.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Popup Demo</title>
+    <meta name="description" content="Popup demo" />
+    <script src="https://aframe.io/releases/1.2.0/aframe.min.js"></script>
+    <script src="components/prism.js"></script>
+    <script src="components/closePopup.js"></script>
+    <script src="components/openPopup.js"></script>
+  </head>
+  <body>
+    <a-scene background="color: #ECECEC" cursor="rayOrigin: mouse">
+      <a-entity
+        id="primaryCamera"
+        position="0 0 9"
+        rotation="0 0 0"
+        camera
+        look-controls
+        wasd-controls
+      ></a-entity>
+      <a-entity id="readRoom" position="0 -100 0">
+        <a-entity
+          id="blackboard"
+          visible="true"
+          geometry="primitive: plane; width: 18   ; height: 15"
+          material="color: #333333; shader: flat; transparent: false"
+        >
+        </a-entity>
+        <a-entity id="returnButton"></a-entity>
```

Comment:
I believe instead of initializing this in JS, (like on lines 47-51), you should be able to do this directly in here, like:
```
<a-entity id="returnButton" close-popup="json_data: <json_data>"></a-entity>
```